website/static/img/databricks_tutorial/images/dbt_project.yml

```diff
@@ -0,0 +1,39 @@
+
+# Name your project! Project names should contain only lowercase characters
+# and underscores. A good package name should reflect your organization's
+# name or the intended use of these models
+name: 'jaffle_shop'
+version: '1.0.0'
```

Comment:
@amychen1776 You may have moved this locally, but I still see it. Is this also OK to remove?

Reply:
Missed this one!